templates/etc/logcheck/logcheck.conf.j2

```diff
@@ -55,13 +55,17 @@ SORTUNIQ=0
 # Controls the base directory for rules file location
 # This must be an absolute path
 
```

Comment:
For now we'll be managing the `README`'s ourselves, but thanks for the suggestion